### ISSUE TITLE
Actions with no icons are not visible on collapsed toolbar

### ISSUE THREAD

**Issue by hvelarde** [plone/Products.CMFPlone#1569]
I have a content type with an action called **Update**; the action is only visible on the expanded toolbar:

![selection_067](https://cloud.githubusercontent.com/assets/341393/14958796/bf7387e0-1062-11e6-8bcd-1e1c275aae34.png)

With the toolbar collapsed the action is only visible when I hover on it:

![selection_068](https://cloud.githubusercontent.com/assets/341393/14958806/cf6b653c-1062-11e6-8acb-64080d4f2ced.png)
![selection_069](https://cloud.githubusercontent.com/assets/341393/14958805/cf5b9abc-1062-11e6-9926-05e4568f4c6e.png)

**Comment by davilima6** [comment]
We should add a sensible default fallback icon in Plone-Fontello (like #1233) and then allow addons to configure the portal_action's `icon_expr` to overwrite it.

We could detect if user input is a character entity, to be used in ::before, or an SVG/PNG, like in #1236.

**Comment by jensens** [comment]
i'd really love to see svg support using actions icon expressions

**Comment by alecpm** [comment]
A default icon has been added in 5.0.x and 5.1, a default content type icon has been added to the toolbar menu as well.

**Comment by alecpm** [comment]
See: 
https://github.com/plone/plone.app.contentmenu/commit/b022808cd607f3776b6ca93f100aab48c9b0c62e
https://github.com/plone/Products.CMFPlone/commit/b3c9e42dda58e9ca94a54ce0d90a55c5e4d64182
